composer.json

```diff
@@ -18,18 +18,18 @@
         }
     ],
     "require": {
-        "php": ">=7.2.0",
```

Comment:
I agree with @kenjis  as 7.2 was already there we should not change the php version. If phpunit was the reason switching to 7.4 we should use yoast polyfill.